ci-automation/vendor-testing/aws.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright (c) 2022 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for the AWS vendor image.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+image_name="ci-${CIA_VERNUM//+/-}"
+aws_instance_type_var="AWS_${CIA_ARCH}_INSTANCE_TYPE"
+aws_instance_type="${!aws_instance_type_var}"
+more_aws_instance_types_var="AWS_${CIA_ARCH}_MORE_INSTANCE_TYPES"
+set -o noglob # there shouldn't be any instance types with asterisks
+              # in it, but…
+more_aws_instance_types=( ${!more_aws_instance_types_var} )
+set +o noglob
+
+vmdk='flatcar_production_ami_vmdk_image.vmdk'
+
+if [[ "${AWS_AMI_ID}" == "" ]]; then
+    if [[ -f "${vmdk}" ]]; then
+        echo "++++ ${CIA_TESTSCRIPT}: using existing ${vmdk} for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+    else
+        echo "++++ ${CIA_TESTSCRIPT}: downloading ${vmdk}.bz2 for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+        copy_from_buildcache "images/${CIA_ARCH}/${CIA_VERNUM}/${vmdk}.bz2" .
+        lbunzip2 "${vmdk}.bz2"
+    fi
+
+    AWS_BUCKET="flatcar-kola-ami-import-${AWS_REGION}"
+    trap 'ore -d aws delete --region="${AWS_REGION}" --name="${image_name}" --ami-name="${image_name}" --file="${vmdk}" --bucket "s3://${AWS_BUCKET}/${board}/"' EXIT
```

Comment:
```suggestion
    trap 'ore -d aws delete --region="${AWS_REGION}" --name="${image_name}" --ami-name="${image_name}" --file="${vmdk}" --bucket "s3://${AWS_BUCKET}/${CIA_VERNUM//+/-}/${board}/"' EXIT
```
Reading the ore upload code I think we have a clash in the s3 URL, and if we don't have it now it's easy to have one depending on the other parameters.